config.yaml

```diff
@@ -3,25 +3,37 @@
 queue_max_size: 10
 
 video_input:
-    camera_name: 0
     worker_period: 1.0  # seconds
-    save_prefix: "log_image"
+    camera_enum: 0  # enum values can be found in camera_factory.py
+    width: 1920
+    height: 1200
+    # For camera_enum=0, use the CV camera config. For camera_enum=1, use the Picamera config
+    camera_config:  # For CV camera (regular cameras, enum 0)
+        device_index: 0
+    # camera_config:  # For Picamera (PiCamera NoIR, enum 1)
+    #     exposure_time: 250  # microseconds
+    #     analogue_gain: 64.0  # Sets ISO, 1.0 for normal, 64.0 for max, 0.0 for min
+    #     contrast: 1.0  # Contrast, 1.0 for nomral, 32.0 for max, 0.0 for min
+    #     lens_position: null  # Focal length, 1/m (0 for infinity, null for auto focus)
+    save_prefix: "log_image"  # Leave as empty string to not log any images
```

Comment:
Update this comment. Might be worth adding a boolean option for enabling/disabling logging.